modules/services/clipmenu.nix

```diff
@@ -28,6 +38,9 @@ in {
 
     home.packages = [ cfg.package ];
 
+    home.sessionVariables =
+      mkIf (cfg.launcher != null) { CM_LAUNCHER = cfg.launcher; };
```

Comment:
If this environment variable is needed by the daemon, then it should be set in the Service Environment; otherwise, it won't be available unless imported (`systemctl --user import-environment CM_LAUNCHER`).

The benefit of setting it there is that it should cause a restart of the service if the value changes.

Reply:
The environment variable is not needed by the daemon but only by the CLI launcher. It can be changed without restarting the service.